src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -969,4 +975,4 @@ public static void onAdvancementProgressedEvent(Player player, Advancement progr
     {
         MinecraftForge.EVENT_BUS.post(new AdvancementProgressEvent(player, progressed, advancementProgress, criterion, progressType));
     }
-}
+}
```

Comment:
i can't re-add it, for some reason when i open a file last empty line got deleted... even if i readd it manually, on ModSorter is more easier because that file doesn't require changes, but on modified classes is imposible on my side